### ISSUE TITLE
Do noop and identity really belong into the public exports of rxjs?

### ISSUE THREAD

**Issue by felixfbecker** [ReactiveX/rxjs#5407]
## Bug Report

**Current Behavior**
This is a minor issue, but every time I type `noop` or `identity` in my projects I get a VS Code autocomplete suggestion to auto-import it from `rxjs`. This feels weird to me, because those functions don't really have anything to do with Observables. My natural place to import them from would be `lodash`, but instead `rxjs` is always suggested first.

This seems to have been very intentional though: https://github.com/ReactiveX/RxJS/commit/810c4d0

**Expected behavior**
These to just be internal functions (unless there's a good reason I am missing).

**Environment**
- RxJS version: 6.5.5

**Possible Solution**
Not export in index.ts

**Additional context/Screenshots**

![image](https://user-images.githubusercontent.com/10532611/80275562-6432c080-86e2-11ea-8520-aadf6b728869.png)

**Comment by kwonoj** [comment]
Makes sense to me too, not sure if it's worth we reexport these utilities.

**Comment by benlesh** [comment]
Hi @felixfbecker — thank you for taking the time to open this issue and for sharing your thoughtful feedback with the RxJS project. I’m sorry it took us so long to respond. Given how much time has passed and the lack of recent activity, I’m going to close this for now so the tracker reflects the work we’re likely to act on. If this is still relevant on a current RxJS release, please feel welcome to open a new issue with updated context and a reproduction. Thank you again for your contribution and patience.

— Ben
